src/locale/locales/en.json

```diff
@@ -31,6 +31,14 @@
     "value": "Last page",
     "comment": "tooltip and aria-label for the last page button. (tew 210929)"
   },
+  "SNTable.Pagination.RowsPerPageChange": {
+    "value": "Rows per page has changed (Rows: {0}). Now showing the first page.",
```

Comment:
@tklkan I think your other suggestion was better `Rows per page has changed to {0}. Now showing the first page.`. Or would that have the same problem?

Reply:
Actually that one should work. Let me update this.